packages/components/src/select/Select.module.css

```
@@ -34,14 +34,18 @@
   color: --text-primary;
   display: flex;
   justify-content: flex-end;
-  height: 3rem;
+  min-height: --size-150;
   align-items: center;
   padding-left: 1rem;
   padding-right: 1rem;
```

```suggestion
  padding-right: --size-70;
```

Så synkar vi utseendet mellan combobox och select

---

fixat